tools/modules/framework/interface_categories.sh

```
@@ -0,0 +1,166 @@
+framework_options+=(
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)